docs/src/starknet/multicall.md

```diff
@@ -26,21 +29,21 @@ unique = false
 
 [[call]]
 call_type = "invoke"
-contract_address = "0x38b7b9507ccf73d79cb42c2cc4e58cf3af1248f342112879bfdf5aa4f606cc9"
+contract_address = "map_contract"
 function = "put"
 inputs = ["0x123", "234"]
 ```
 
 After running `sncast multicall run --path file.toml`, a declared contract will be first deployed, and then its function `put` will be invoked.
 
+> 📝 **Note**
+> You can add property `id` to the deploy call specification and then use it as a `contract_address` when declaring an invoke call 🔥
```

Comment:
Maybe something like "In the example above, we have used `id` property in `deploy` call (...)` ?